extensions/wikia/Email/Email.i18n.php

```diff
@@ -16,6 +16,10 @@
 	'emailext-unfollow-text' => 'No longer interested in receiving these updates? Click [$1 here] to unfollow $2 on {{SITENAME}}.',
 	'emailext-salutation' => 'Hi $1,',
 	'emailext-wikia-home-url' => 'http://www.wikia.com/explore',
+	'emailext-mobile-application-footer-1' => 'Take your favorite fandoms with you and never miss a beat.',
+	'emailext-mobile-application-footer-2' => 'Check out our Fandom Community apps!',
+	'emailext-mobile-application-ios-badge' => 'App-store-badge-en.svg',
```

Comment:
How about we move the assets configuration to a separate config file / structure? So that we do not confuse translators when pushing these i18n files to crowdin.